pkg/kube/inject/testdata/inject/hello-ignore.yaml.injected

```diff
@@ -14,20 +14,10 @@ spec:
   template:
     metadata:
       annotations:
-        prometheus.io/path: /stats/prometheus
```

Comment:
Everything is removed since this is supposed to be ignored by `sidecar.istio.io/inject: "false"` which was fixed